src/frontend/App/hooks/useVerdierForBrev.ts

```diff
@@ -6,6 +6,14 @@ import { IBeløpsperiode, IBeregningsperiodeBarnetilsyn } from '../typer/vedtak'
 enum EBehandlingFlettefelt {
     fomdatoInnvilgelseForstegangsbehandling = 'fomdatoInnvilgelseForstegangsbehandling',
     tomdatoInnvilgelseForstegangsbehandling = 'tomdatoInnvilgelseForstegangsbehandling',
+    fomdatoInnvilgelse = 'fomdatoInnvilgelse',
+    tomdatoInnvilgelse = 'tomdatoInnvilgelse',
+    fomdatoInnvilgelseBarnetilsyn = 'fomdatoInnvilgelseBarnetilsyn',
+    tomdatoInnvilgelseBarnetilsyn = 'tomdatoInnvilgelseBarnetilsyn',
+    fomdatoInnvilgelseSkolepenger = 'fomdatoInnvilgelseSkolepenger',
+    tomdatoInnvilgelseSkolepenger = 'tomdatoInnvilgelseSkolepenger',
+    fomdatoRevurderingBT = 'fomdatoRevurderingBT',
+    tomdatoRevurderingBT = 'tomdatoRevurderingBT',
```

Comment:
Burde vi kunne gjenbruke 1 felt for alle fom/tom-datoer her? 👀 
Eks `innvilgetFra` / `innvilgetTom`

Reply:
Vi kunne gjenbrukt dem, men de kan ha forskjellige visningsnavn i brevmenyen, så det får nesten være en funksjonell avklaring om vi kan bruke det samme feltet.

Reply:
Ah, ser den! Det burde man nesten kunne bruke ut fra stønadstypen på fagsaken?

Reply:
Ikke helt sikker på jeg skjønner hva du mener 😅  Men visningen av flettefelt i brevmenyen er generisk, så å implementere egen visning av flettefelt for vedtaksdatoer basert på stønadstype tror jeg er mer jobb enn å ha 8 forskjellige flettefelter :)

Reply:
Ah, trodde vi viste titler og sånt fra oss, men ser nå at det ikke er sånn 👍